src/fastmath/ml/regression.clj

```diff
@@ -1246,3 +1269,61 @@
      {:dose ##Inf
       :p p
       :stderr ##Inf})))
+
+(defn- residuals-analysis
+  "Summary statistics used for printing residuals with model"
+  [residuals]
+  [{:min (apply min residuals)
+    :1q (stats/quantile residuals 0.25)
+    :median (stats/median residuals)
+    :3q (stats/quantile residuals 0.75)
```

Comment:
You can get quantiles with one call `(stats/quantiles residuals [0.25 0.5 0.75])`